packages/react-native-bpk-theming/src/BpkThemeAttributes.js

```diff
@@ -89,6 +89,11 @@ export type Props = {
  *
  */
 const BpkThemeAttributes = (props: Props) => {
+  // eslint-disable-next-line no-console
```

Comment:
I think we have a helper for this warning, but maybe that was only for deprecated proptypes